integration_test/gce/gce_testing.go

```diff
@@ -1109,18 +1130,30 @@ func attemptCreateInstance(ctx context.Context, logger *log.Logger, options VMOp
 		return nil, fmt.Errorf("attemptCreateInstance() could not construct valid labels: %v", err)
 	}
 
-	imageOrImageFamilyFlag := "--image-family=" + vm.Platform
+	imageOrImageFamilyFlag := "--image=" + vm.Platform
```

Comment:
Would slightly prefer for this to pull from options.Image instead of vm.Platform because that's IMO less confusing.

Same thing with lines 1136 and 1139 (options.Platform this time instead of vm.Platform).